Sources/TuistSupport/SwiftPackageManager/SwiftPackageManagerController.swift

```diff
@@ -70,6 +81,46 @@ public final class SwiftPackageManagerController: SwiftPackageManagerControlling
         return try decoder.decode(PackageInfo.self, from: data)
     }
 
+    public func buildFatReleaseBinary(
+        packagePath: AbsolutePath,
+        product: String,
+        buildPath: AbsolutePath,
+        outputPath: AbsolutePath
+    ) throws {
+        let buildCommand: [String] = [
+            "swift", "build",
+            "--configuration", "release",
+            "--disable-sandbox",
+            "--package-path", packagePath.pathString,
+            "--product", product,
+            "--build-path", buildPath.pathString,
+            "--triple",
+        ]
```

Comment:
don't we need also the `-enable-library-evolution`?

Reply:
this function builds binaries only and this flag is applicable only for frameworks, so I believe it should not be necessary here.